docs/reference/ssl/swarm-with-traefik.md

```diff
@@ -0,0 +1,319 @@
+# SSL on Swarm with Traefik
+
+To completely secure your OpenFaaS installation, you need SSL.  On Swarm, you can do this easily with [Traefik][traefik] and [Let's Encrypt][letsencrypt].  Traefik is is reverse proxy that comes with SSL support via Let's Encrypt. In this tutorial we will show you how to deploy OpenFaaS with Traefik.
+
+
+## Create an A record
+
+If your domain is `.domain.com` then create an A record using your DNS administration panel such as `gateway.domain.com` or `openfaas.domain.com`. The required steps will vary depending on your domain provider and your cluster provider. For example; [on Google Cloud DNS](https://cloud.google.com/kubernetes-engine/docs/tutorials/configuring-domain-name-static-ip) or [with Route53 using AWS](https://kubernetes.io/docs/setup/custom-cloud/kops/#2-5-create-a-route53-domain-for-your-cluster).
```

Comment:
Query whether we should call out specific CSPs as examples if we aren't mentioning _all_ of them.

Reply:
This follows the statement in the k8s documentation. I will gladly add more, of course _all_ is not possible

Reply:
CSP?

Reply:
Cloud service provider